Makefile

```diff
@@ -9,8 +9,10 @@ help:
 	@echo '  test - Run all tests.'
 	@echo 'Releasing packages:'
 	@echo '  release - Bump versions, commit and push changes to the repository and release gems. Requires clean repository and VERSION set.'
+	@echo 'Deploying applications:'
+	@echo '  deploy_staging - Deploy an application to the staging pipeline. Requires heroku login and HEROKU_APP_NAME set.'
```

Comment:
`Requires heroku login and HEROKU_APP_NAME set.`
If I am not wrong `HEROKU_APP_NAME` is actually set by `deploy_staging.sh` so setting it before launching the script will have no effect?